schema/derive/tests/ui_pass/derive_into_schema.rs

```diff
@@ -13,17 +13,34 @@ pub struct Foo2<T, K> {
 
 #[derive(IntoSchema)]
 pub struct Foo3<T: Clone> {
-   _value1: T,
+    _value1: T,
```

Comment:
One baffling decision in `rustfmt` is the fact that it doesn't handle whitespace. A lot of the time the whole reason one wants to have an autoformatter is to ensure that the pesky spaces are inserted.